drake/examples/Acrobot/test/paramEstSyntheticData.m

```diff
@@ -1,26 +1,99 @@
 function paramEstSyntheticData
 
-oldpath = addpath(fullfile(pwd,'..'));
```

Comment:
I took it out because Andres and I found that the line created errors with adding spotless to the environment path.  Did you know anything about that and if this new method fixes it?